### ISSUE TITLE
Remove history items

### ISSUE THREAD

**Issue by tuomas2** [AndBible/and-bible#374]
Related to #231 

From Darryl
> Would it be possible to include a feature to remove selected items from the history, in order to clean it up after meandering readings? So a long press on the history list would activate a multi-select context to remove selected history items.

**Comment by agrogers** [comment]
I agree that the history does get filled with wanderings. But i can't see how you could avoid that. And i dont think i would ever take the time to clean it up.

Perhaps trying to indent or bold the history items somehow. So a verse that you choose from the verse selector is shown in bold or as a parent. Then all the scrolling and jumping around is indented or shown in smaller print. That would make it easier to quickly scroll back up to the main verse you searched for previously.

If that concept had potential, then the history could be done as a real tree with the 'wandering verses' collapsed under the main verses.

**Comment by agrogers** [comment]
I suggest we delay this one to after v3.4 release
